src/plugins/Sharesite/Freesite.java

```diff
@@ -76,7 +76,22 @@ public Freesite(int uniqueKey) {
 			this.css= "";
 		}
 
+		try {
+			InputStream is = Plugin.class.getClassLoader().getResourceAsStream(texttemplate);
```

Comment:
Thank you for the catch! I now switched do try-with-resources for both the text loading and the css loading for both the input stream and the BufferedReader.